babycenter/static/css/_layout.scss

```diff
@@ -2,6 +2,11 @@
   padding: 8px 16px;
 }
 
+.l-indent--small {
```

Comment:
If they're concerned about the height of the `headings,` which surprises me, then I think you should modify `l-indent` to be `6px 16px` instead of creating a new class.

`line-height: 1.2` seems like a magic number to me here?